vissl/hooks/log_hooks.py

```diff
@@ -173,42 +173,35 @@ def on_update(self, task: "tasks.ClassyTask") -> None:
                 avg_time = sum(batch_times) / len(batch_times)
 
                 eta_secs = avg_time * (task.max_iteration - iteration)
-                eta_string = str(datetime.timedelta(seconds=int(eta_secs)))
                 if isinstance(task.optimizer.options_view.lr, set):
                     lr_val = list(task.optimizer.options_view.lr)
                 else:
                     lr_val = round(task.optimizer.options_view.lr, 5)
                 batch_time = int(1000.0 * avg_time)
                 rank = get_rank()
-                log_data = {
-                    "Rank": rank,
-                    "ep": train_phase_idx,
-                    "iter": iteration,
-                    "lr": lr_val,
-                    "loss": loss_val,
-                    "btime(ms)": batch_time,
-                    "eta": eta_string,
-                    "peak_mem(M)": peak_mem_used,
-                }
+                evt_stg.put_scalars(
+                    rank=rank,
+                    epoch=train_phase_idx,
```

Comment:
can you insert pdb here to verify the value of `train_phase_idx` when it changes and if this is properly logged to storage too?